modules/images-other-jenkins-env-var.adoc

```diff
@@ -0,0 +1,114 @@
+option contains space characters, escape them with a backslash. Example
+settings: `-Dfoo -Dbar`; `-Dfoo=first\ value -Dbar=second\ value`.
+
+|`JENKINS_OPTS`
+|Specifies arguments to Jenkins.
+
+|`INSTALL_PLUGINS`
+|Specifies additional Jenkins plug-ins to install when the container is first run
+or when `OVERRIDE_PV_PLUGINS_WITH_IMAGE_PLUGINS` is set to `true` (see below).
+Plug-ins are specified as a comma-delimited list of name:version pairs. Example
+setting: `git:3.7.0,subversion:2.10.2`.
+
+|`OPENSHIFT_PERMISSIONS_POLL_INTERVAL` (default: `300000` - 5 minutes)
+|Specifies in milliseconds how often the OpenShift Login plug-in polls
+{product-title} for the permissions associated with each user defined in Jenkins.
+
+|`OVERRIDE_PV_CONFIG_WITH_IMAGE_CONFIG` (default: `false`)
+|When running this image with an {product-title} persistent volume for the Jenkins
+config directory, the transfer of configuration from the image to the persistent
+volume is only done the first startup of the image as the persistent volume is
+assigned by the persistent volume claim creation. If you create a custom image
+that extends this image and updates configuration in the custom image after
+the initial startup, by default it is not copied over, unless you set this
+environment variable to `true`.
+
+|`OVERRIDE_PV_PLUGINS_WITH_IMAGE_PLUGINS` (default: `false`)
+|When running this image with an {product-title} persistent volume for the Jenkins
+config directory, the transfer of plugins from the image to the persistent
+volume is only done the first startup of the image as the persistent volume is
+assigned by the persistent volume claim creation. If you create a custom image
+that extends this image and updates plugins in the custom image after
+the initial startup, by default they are not copied over, unless you set this
+environment variable to `true`.
+
+|`ENABLE_FATAL_ERROR_LOG_FILE` (default: `false`)
+|When running this image with an {product-title} persistent claim for the Jenkins
+config directory, this environment variable allows the fatal error log file to
+persist when a fatal error occurs. The fatal error file is saved at
+`/var/lib/jenkins/logs`.
+
+|`NODEJS_SLAVE_IMAGE`
+|Setting this value overrides the image used for the default NodeJS agent pod configuration.
+The default NodeJS agent pod uses `docker.io/openshift/jenkins-agent-nodejs-8-centos7` or
+`registry.redhat.io/openshift4/jenkins-agent-nodejs-8-rhel7` depending whether you are running
+the CentOS or RHEL version of the Jenkins image. This variable must be set before Jenkins starts
+the first time for it to have an effect.
+
+|`MAVEN_SLAVE_IMAGE`
+|Setting this value overrides the image used for the default maven agent pod configuration.
+The default maven agent pod uses `docker.io/openshift/jenkins-agent-maven-35-centos7` or
+`registry.redhat.io/openshift4/jenkins-agent-maven-35-rhel7` depending whether you are running
```

Comment:
From 4.x, the default maven agent image in jenkins server is image-registry.openshift-image-registry.svc:5000/openshift/jenkins-agent-maven:latest . There is related imagestreamtag jenkins-agent-maven under openshift project